options.html

```diff
@@ -173,7 +173,7 @@ <h1>About</h1>
 
 				<div class="content">
 					<div class="image-container">
-						<img src="img/icon_32_kk.png" width="32" height="32" alt="Add-ons for NeoGAF">
+						<img src="img/icon_32.png" width="32" height="32">
```

Comment:
What about using the 128x128 version here? It does make a difference in my retina display

Reply:
Done in 38ee7b7

Reply:
<table><tr><td>
<img width="265" alt="Screenshot 2020-01-17 at 11 59 48" src="https://user-images.githubusercontent.com/16718039/72607482-12c7a800-3921-11ea-9215-9950c6a36938.png"></td><td>
vs</td>
<td>
<img width="247" alt="Screenshot 2020-01-17 at 12 00 00" src="https://user-images.githubusercontent.com/16718039/72607488-165b2f00-3921-11ea-8ac9-bf48ac1b69d6.png">
</td></tr></table>